chat-plugins/animeandmanga.js

```diff
@@ -0,0 +1,121 @@
+//Anime and Manga search by SilverTactic (Siiilver)
+//This script searches for an anime/manga based on a given query and displays (usually relevant) results
+//It uses the database/api of anilist
+'use strict';
+
+//DO NOT CHANGE THESE
+const ID = "silvy-46dh7";
+const SECRET_ID = "Mhz6vrcHF8qPS8v2mA7j3vaXl";
+
+let nani = require('nani').init(ID, SECRET_ID);
+
+if (!global.amCache) global.amCache = {anime:{}, manga:{}};
```

Comment:
Yeah, it needs to be a global. I kept it this way instead of using the <code>Gold</code> global because that's what the script used to begin with